src/components/managementPanel/ManagementPanel.jsx

```diff
@@ -1,32 +1,32 @@
-import {useEffect, useState} from 'react';
-import {Rectangle, Tooltip} from 'react-leaflet';
-import {useTranslation} from 'react-i18next';
-import {useDispatch} from 'react-redux';
+import { useEffect, useState } from 'react';
+import { Rectangle, Tooltip } from 'react-leaflet';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
 import InputLabel from '@material-ui/core/InputLabel';
 import Select from '@material-ui/core/Select';
 import MenuItem from '@material-ui/core/MenuItem';
 import FormControl from '@material-ui/core/FormControl';
 import Button from '@material-ui/core/Button';
-import {CircularProgress} from '@material-ui/core';
+import { CircularProgress } from '@material-ui/core';
 
 import api from '../../api/apiInstance';
-import {basicHeaders} from '../../config/apiConfig';
+import { basicHeaders } from '../../config/apiConfig';
 import H3Title from '../customs/H3Title';
 import colors from '../../stylesheets/config/colors.module.scss';
 import ManagementPanelMap from './ManagementPanelMap';
 import Dashboard from '../Dashboard';
 import RoleAssignmentPanel from './RoleAssignmentPanel';
 import H4Title from '../customs/H4Title';
-import {NotificationActions} from '../../redux/actions/notificationActions';
+import { NotificationActions } from '../../redux/actions/notificationActions';
 
 import '../../stylesheets/managementPanel.scss';
 
 const NONE = 'none';
-const CURRENT_LOCATION = {lat: 50.29, lng: 19.01};
+const CURRENT_LOCATION = { lat: 50.29, lng: 19.01 };
 const ZOOM = 9;
```

Comment:
as above, let's push it into separate file for constants (we do not have any at present)